<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>astm_middleware</class>
 <widget class="QWidget" name="astm_middleware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyzer - ASTM Middleware</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Icons/Icons.qrc">
    <normaloff>:/Main/port.png</normaloff>:/Main/port.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_start_listener">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Start Listener</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons/Icons.qrc">
         <normaloff>:/Buttons/icons8-connected-94.png</normaloff>:/Buttons/icons8-connected-94.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stop_listener">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Stop Listener</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons/Icons.qrc">
         <normaloff>:/Buttons/icons8-disconnected-94.png</normaloff>:/Buttons/icons8-disconnected-94.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_settings">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
       <property name="icon">
        <iconset resource="../../Icons/Icons.qrc">
         <normaloff>:/Buttons/icons8-settings-375.png</normaloff>:/Buttons/icons8-settings-375.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="table_layout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QGroupBox" name="gb_sqlite_view_data_table">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="sqlite_db_table_layout">
        <item>
         <widget class="QTableWidget" name="sqlite_data_table">
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <column>
           <property name="text">
            <string>DateTime</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Analyzer</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Sample ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Message</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_astm_message">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>ASTM Message from the Analyzer</string>
       </property>
       <layout class="QVBoxLayout" name="astm_msg_layout">
        <item>
         <widget class="QTextEdit" name="astm_msg_textEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
